automated_analysis.py

```diff
@@ -559,4 +559,48 @@ def set_survey_percentages(survey_counts, total_survey_counts):
         log.info("Skipping generating a map of per-constituency theme participation because "
                  "`generate_constituency_theme_distribution_maps` is set to False")
 
+    # Export raw messages labelled with accountability and safeguarding labels.
+    log.info("Exporting success story raw messages for each episode...")
```

Comment:
```suggestion
    log.info("Exporting accountability raw messages for each episode...")
```

Great that this looks it can form a standardised function though :)